### PR TITLE
Restructure

### PR DESCRIPTION
**Pull request by geet084**
many things were addressed, below are the highlights:
- fix dataset import to have extra cities for NY & LA / matching array length for carousel
- refactor App for increased readability
- refactor Card to be functional & child of Popup
- add City to be a functional child of Popup
- refactor Container to be parent of Popup
    - refactor from index position to a dynamic class name indexes/identifiers
    - add refactored carousel to match dynamic indexing/identifiers
- add Popup to be child of Container and parent of Card & City 
- refactor TeamIcon to match styling of other stateless components

@b3an5 
@ericfitzsimons451 
@robbiejaeger @brittanystoroz @thatpamiam

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -36,40 +37,49 @@ class App extends Component {
 
     this.getData('cities').then((city) => {
       this.setState({
-        cities: city.cities
+        cities: this.matchCitiesToTeams(city.cities)
       })
     }).catch(errrr => console.log(errrr))
   }
 
+  matchCitiesToTeams(cities) {
```
[the remark this answers is not in the document]
Comment (geet084) [reply src/App.js]: I'm sure that we could, however knowing that we will almost always have to manage the dataset we get into something that we want, I feel like this path will be more likely than getting a "perfect" dataset to work from

[the remark this answers is not in the document]
Comment (ericfitzsimons451) [reply src/App.js]: Tru dat

**Comment by b3an5** [comment]
full afc and nfc dropdown is broken as well as the css for the card

**Comment by geet084** [comment]
@b3an5 
That's my fault, I tried to refactor it to be cleaner and missed the "includes" portion for the AFC/NFC conference.
Nice catch!! all fixed now

**Pull request merged**